ddp-workspace/projects/ddp-pancan/src/app/components/anchors-page/anchors-page.component.html

```diff
@@ -0,0 +1,43 @@
+<main class="page">
+    <h2 class="title static-page-title">{{source + '.Title' | translate}}</h2>
+
+    <nav class="anchors">
+        <ul>
+            <li *ngFor="let section of (source + '.Sections') | translate; let i = index;" class="anchor">
+                <a [routerLink]="[routerLink]"  [fragment]="'section' + i"
+                   (click)="scrollToAnchor('section' + i)">{{section.Title}}
```

Comment:
should anchorLabel be used here as well? (and on the line above)

Reply:
Sure, fixed.